.env

```diff
@@ -0,0 +1 @@
+DB_ENV=development
```

Comment:
I got thrown off yesterday while looking at your code with you--this _is_ the correct variable setting here, since you have your cross-env command in your test script overriding it to be testing